modules/launcher/src/test/java/org/mule/runtime/module/launcher/log4j2/ArtifactAwareContextSelectorTestCase.java

```diff
@@ -164,6 +171,20 @@ public void returnsMuleLoggerContextForInternalArtifactClassLoader() {
     assertThat(serviceCtx, sameInstance(systemContext));
   }
 
+  @Test
+  public void returnsParentContextForPolicyClassloader() throws MalformedURLException {
+    ClassLoader childClassLoader = new URLClassLoader(new URL[0], regionClassLoader);
+    PolicyTemplateDescriptor policyTemplateDescriptor = new PolicyTemplateDescriptor(POLICY_TEMPLATE_NAME, empty());
+    RegionClassLoader policyRegionClassLoader = spy(getPolicyRegionClassLoader(policyTemplateDescriptor));
+    MuleArtifactClassLoader policyClassLoader = getPolicyArtifactClassLoader(policyTemplateDescriptor, policyRegionClassLoader);
+
+    LoggerContext appCtx = selector.getContext("", childClassLoader, true);
+    LoggerContext policyCtx = selector.getContext("", policyClassLoader, true);
+
+    assertThat(policyCtx, instanceOf(MuleLoggerContext.class));
```

Comment:
this is testing implementation, not behavior. the only assertion with value is the one below